tests/conftest.py

```diff
@@ -10,29 +10,35 @@
 
 GIT_ROOT = os.path.join(os.path.abspath(os.path.dirname(__file__)), '..')
 
-
 class HlwmBridge:
 
     HC_PATH = os.path.join(GIT_ROOT, 'herbstclient')
 
-    def __init__(self,display):
+    def __init__(self,display,hlwm_process):
```

Comment:
Not sure if you leave them out on purpose, but for what it's worth, PEP 8 prescribes spaces after these commas (there are more occurrences within this change)

Reply:
I'll change it!